docs/UserGuide.md

```diff
@@ -73,6 +73,7 @@ If you can type fast, Tutor's pet can get your class management tasks done faste
 | **Add Class**                   | `add-class n/CLASS_NAME`<br> e.g., `add-class n/CS2103T Tutorial T10`                                                                       |
 | **List All Classes**                   | `list-class`                   |
 | **List Students In A Class**    | `list-students c/INDEX`<br> e.g., `list-students c/3`                                                                                       |
+| **List All Students And Classes** | `list`
```

Comment:
Can you help organise/center the fencing "|" for the userguide table as well? I remembered centering it awhile back during v1.1 but seems like some of the fences got out of position again